lms/djangoapps/verify_student/views.py

```diff
@@ -1175,15 +1179,69 @@ def post(self, request, course_id, checkpoint_name):
                 EVENT_NAME_USER_SUBMITTED_INCOURSE_REVERIFY, user.id, course_id, checkpoint_name
             )
 
-            return HttpResponse()
+            redirect_url = self._get_redirect_url(course_id, location)
+            return JsonResponse({
+                'url': redirect_url
+            })
+        except Http404 as e:
+            log.exception("Invalid location during photo verification.")
+            return HttpResponseBadRequest(_(e.message))
         except IndexError:
             log.exception("Invalid image data during photo verification.")
             return HttpResponseBadRequest(_("Invalid image data during photo verification."))
         except Exception:  # pylint: disable=broad-except
-            log.exception("Could not submit verification attempt for user {}.").format(request.user.id)
+            log.exception("Could not submit verification attempt for user {}.".format(request.user.id))
             msg = _("Could not submit photos")
             return HttpResponseBadRequest(msg)
 
+    def _get_redirect_url(self, course_id, location):
```

Comment:
since it's very much coupled to courseware, one suggestion would be creating a helper module (e.g. `url_helpers`) in courseware and putting the function there.   you'd want to avoid having that module import anything else in courseware, to avoid a circular import situation.